lib/commons/aria/roles.js

```diff
@@ -21,6 +21,19 @@ aria.isValidRole = function(
 	return allowAbstract ? true : roleDefinition.type !== 'abstract';
 };
 
+/**
+ * Check if a given role is unsupported
+ * @method isUnsupportedRole
+ * @memberof axe.commons.aria
+ * @instance
+ * @param {String} role The role to check
+ * @return {Boolean}
+ */
+aria.isUnsupportedRole = function(role) {
```

Comment:
I don't think we need a new method for this. I don't expect we'll have another use for it. Also, if we do add a method to this, it should go into its own file and have unit tests for that. Makes the repo easier to navigate that way.

Reply:
My reasoning was that the fix to pass on invalid roles when checking for unsupported roles inside `isValidRole` created two separate code paths in the function. At that point I figured they should be 2 separate functions to keep them doing one thing each.